test/functional/fixtures/api/es-next/navigate-to-and-test-page/testcafe-fixtures/navigate-to-test.js

```diff
@@ -1,11 +1,41 @@
+import path from 'path';
+import { Selector, ClientFunction } from 'testcafe';
+
 fixture `NavigateTo`
     .page `http://localhost:3000/fixtures/api/es-next/navigate-to-and-test-page/pages/index.html`;
 
+const pageHeader  = Selector('h1');
+const getLocation = ClientFunction(() => window.location.toString());
+
 test('Navigate to another page', async t => {
     await t.navigateTo('navigation.html')
         .click('#button');
 });
 
+test('Navigate to relative file page', async t => {
+    await t
+        .navigateTo('../pages/relative.html')
```

Comment:
It shouldn't resolve to `file` protocol. If we are on HTTP or HTTPS page we should resolve relative to current URL.